src/SplitPageLayout/SplitPageLayout.test.tsx

```diff
@@ -17,7 +17,8 @@ describe('SplitPageLayout', () => {
   })
 
   it('renders default layout', () => {
-    const spy = jest.spyOn(console, 'warn').mockImplementation(() => {})
+    // Disabling them for now because we are not expecting them in the tests.
+    // const spy = jest.spyOn(console, 'warn').mockImplementation(() => {})
```

Comment:
Oooh yeah, that's not nice. Can we create set a variable in environment just like `__DEV__` called `__PRIMER_TEST__` (or process.env.primer) so that we can keep testing this in primer's test suite but not in dotcom's?

Reply:
Just for the records, as we settled for the TS `@deprecated` annotation for the time being, we are gonna worry about the warnings later 😄